### ISSUE TITLE
Feature request: add link type `/dns/<domain name>`

### ISSUE THREAD

**Issue by buckle2000** [ipfs/notes#348]
How about we change `/ipns/<domain name>` into `/dns/<domain name>`?
The syntax `/ipns/<domain name>` is really confusing.
Related: https://github.com/ipfs/ipfs-update/issues/80#issuecomment-370313268

**Comment by lidel** [comment]
@lgierth do you remember the story behind `/dns/{website_with_dnslink}` ? 
(why we went with  `/ipns/{website_with_dnslink}` etc)

**Comment by swedneck** [comment]
why not use `/dnslink` instead of `/dns`? Since multiformats/go-multiaddr-dns/pull/17 is a thing it seems quite confusing to use `/dns` to resolve dnslink, and `/dnslink` is intuitive to me at least.

**Comment by lidel** [comment]
I agree: `/dnslink` is clear and removes ambigouity about the purpose of the namespace.

It also matches existing `/dnsaddr` quite nicely.

**Comment by momack2** [comment]
@Stebalien @aschmahmann - solving old problems?

**Comment by Stebalien** [comment]
Solving old problems...

/dnslink is the correct way forward.
